keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -158,18 +159,16 @@ with ipBlock rules that select external traffic.
 ## Proposal
 
 In order to achieve the three primary broad use cases for a cluster admin to
-secure K8s clusters, we propose to introduce the following resource
+secure K8s clusters, we propose to introduce the following resources
 under `policy.networking.k8s.io` API group:
 - AdminNetworkPolicy
+- BaselineAdminNetworkPolicy
 
 ### AdminNetworkPolicy resource
 
-An AdminNetworkPolicy (ANP) resource will help the administrators:
-1. Set strict security rules for the cluster, i.e. a developer CANNOT override
-these rules by creating NetworkPolicies that applies to the same workloads as the
-AdminNetworkPolicy does.
-2. Set baseline security rules that describes default connectivity for cluster
```

Comment:
wait we are still doing this with baseline adminnetworkpolicies right? Why did you change this bit